dtformats/indx_directory_entry.yml

```diff
@@ -0,0 +1,158 @@
+name: index_record
+type: format
+description: Index Directory Entry
+urls: ["https://github.com/libyal/libfsntfs/blob/83c2f4ce3d16b5535eae9de767adc93fff724004/documentation/New%20Technologies%20File%20System%20(NTFS).asciidoc#index"]
+metadata:
+  authors: ['Joachim Metz <joachim.metz@gmail.com>', 'Juan Leaniz <juan.leaniz@gmail.com']
```

Comment:
for multiple entries use:
```
authors:
-  Juan Leaniz <juan.leaniz@gmail.com'
```

However you can remove me at this point since I did not author this file